### PR TITLE
Addressing text review comments

### PR DESCRIPTION
**Pull request by anuchandy**
Addressing https://github.com/MSOpenTech/azure-xplat-cli-pr/issues/376

### CODE REVIEW

**Comment by anuchandy** [comment]
@ruslangabitov JFYI we asked help from CONTENT DEVELOPER to review this.

File: lib/commands/arm/vm/vm._js [hunk 1]
```diff
@@ -407,7 +407,7 @@ exports.init = function (cli) {
     });
 
     extension.command('get [resource-group] [vm-name]')
-      .description($('Get VM extensions'))
+      .description($('Get extensions installed VM in a resource group'))
```
Comment (ruslangabitov) [inline lib/commands/arm/vm/vm._js]: Shouldn't it be 'Get extensions installed on ...' ?

Comment (anuchandy) [inline lib/commands/arm/vm/vm._js]: right, will fix this.

File: lib/commands/arm/network/network._js [hunk 2]
```diff
@@ -472,7 +472,7 @@ exports.init = function (cli) {
     });
 
   loadBalancerAddressPool.command('delete [resource-group] [lb-name] [name]')
-    .description($('Deletes an address pool from load balancer within a resource group'))
+    .description($('Delete an address pool from load balancer in a resource group'))
```
Comment (dlepow) [inline lib/commands/arm/network/network._js]: Suggest "...from *a* load balancer..."

File: lib/commands/arm/network/network._js [hunk 3]
```diff
@@ -492,7 +492,7 @@ exports.init = function (cli) {
     .description($('Commands to manage load balancer rules'));
 
   loadBalancerRule.command('create [resource-group] [lb-name] [name]')
-    .description($('Adds a new load balancing rule to the list of load balancer rules within a resource group'))
+    .description($('Add a load balancing rule to load balancer in a resource group'))
```
Comment (dlepow) [inline lib/commands/arm/network/network._js]: Suggest "... to *a* load balancer..."

File: lib/commands/arm/network/network._js [hunk 4]
```diff
@@ -632,7 +632,7 @@ exports.init = function (cli) {
 
   inboundNatRule.command('delete [resource-group] [lb-name] [name]')
     .usage('[options] <resource-group> <lb-name> <name>')
-    .description($('Deletes an existing load balancing inbound NAT rule configured for the load balancer within a resource group'))
+    .description($('Delete a load balancing inbound NAT rule from load balancer in a resource group'))
```
Comment (dlepow) [inline lib/commands/arm/network/network._js]: "...from *a* load balancer..."

File: lib/commands/arm/network/network._js [hunk 5]
```diff
@@ -910,7 +910,7 @@ exports.init = function (cli) {
     .description($('Commands to manage inbound rules of the network interface'));
 
   nicInboundRule.command('add [resource-group] [name]')
-    .description($('Adds a inbound NAT rule to the NIC within a resource group'))
+    .description($('Add a inbound NAT rule to a NIC in a resource group'))
```
Comment (dlepow) [inline lib/commands/arm/network/network._js]: "Add *an* inbound NAT rule..."

File: lib/commands/arm/network/network._js [hunk 6]
```diff
@@ -1116,7 +1116,7 @@ exports.init = function (cli) {
     });
 
   nsgRules.command('delete [resource-group] [nsg-name] [name]')
-    .description($('Deletes a rule in a network security group within a resource group'))
+    .description($('Deletes a rule in a network security group in a resource group'))
```
Comment (dlepow) [inline lib/commands/arm/network/network._js]: "*Delete* a rule..."

File: lib/commands/arm/network/network._js [hunk 7]
```diff
@@ -1245,7 +1245,7 @@ exports.init = function (cli) {
     });
 
   dnsRecordSet.command('set [resource-group] [dns-zone-name] [name] [type]')
-    .description($('Sets a record set in a DNS zone'))
+    .description($('Set DNS zone record set in a resource group'))
```
Comment (dlepow) [inline lib/commands/arm/network/network._js]: "Set *a* DNS zone record set..."

File: lib/commands/arm/vm/vm._js [hunk 8]
```diff
@@ -206,7 +206,7 @@ exports.init = function (cli) {
     });
 
   vm.command('deallocate [resource-group] [name]')
-    .description($('Shutdown one virtual machine within a resource group and releases the compute resources'))
+    .description($('Shutdown a virtual machine in a resource group and releases the compute resources'))
```
Comment (dlepow) [inline lib/commands/arm/vm/vm._js]: "...and *release* ..."

File: lib/commands/arm/vm/vm._js [hunk 9]
```diff
@@ -343,7 +343,7 @@ exports.init = function (cli) {
     });
 
     disk.command('detach [resource-group] [vm-name] [lun]')
-      .description($('Detaches a data-disk attached to a VM'))
+      .description($('Detaches a data-disk attached to a VM in a resource group'))
```
Comment (dlepow) [inline lib/commands/arm/vm/vm._js]: "*Detach* a data-disk...."

**Pull request merged**